bot/exts/utilities/conversationstarters.py

```diff
@@ -36,6 +36,7 @@ class ConvoStarters(commands.Cog):
     """General conversation topics."""
 
     @commands.command()
+    @commands.cooldown(1, 300, commands.BucketType.channel)
```

Comment:
What if the cool down was only per user?
```suggestion
    @commands.cooldown(1, 300, commands.BucketType.user)
```

Reply:
The reason for a channel cooldown was so that multiple users can't spam the topic command in a single channel